.github/workflows/integration-tests.yml

```
@@ -114,3 +114,13 @@ jobs:
           POSTGRES_DB: postgres_doctrine_test
           POSTGRES_USER: postgres
           POSTGRES_PASSWORD: postgres
+
+      - name: Upload coverage results to Coveralls
+        if: matrix.calculate-code-coverage == true
```

actionlint (1.7.7)

property "calculate-code-coverage" is not defined in object type {php: number; postgres: number}

(expression)